go/cmd/ccql/main.go

```diff
@@ -34,12 +35,13 @@ func main() {
 		osUser = usr.Username
 	}
 
+	osUser = "root"
 	help := flag.Bool("help", false, "Display usage")
 	user := flag.String("u", osUser, "MySQL username")
 	password := flag.String("p", "", "MySQL password")
 	askPassword := flag.Bool("ask-pass", false, "prompt for MySQL password")
 	credentialsFile := flag.String("C", "", "Credentials file, expecting [client] scope, with 'user', 'password' fields. Overrides -u and -p")
-	defaultSchema := flag.String("d", "information_schema", "Default schema to use")
+	schemaList := flag.String("s", "information_schema", "List of Schema to query from.")
```

Comment:
This is a breaking, incompatible change. Please avoid doing this. Find a backwards compatible way.